README.md

```diff
@@ -174,6 +174,26 @@ The difference with `make up-stable` is that command wouldn't start common compo
 make up-blue
 ``` 
 
+### How to run a limited set of Open Kilda modules
+
+If you do not need to run some components (e.g. GUI, any topology, etc.) you can use 
+[docker-compose.yaml](confd/vars/docker-compose.yaml) file.
+
+This file contains a list of kilda modules. If you decided not to run some of them you need to do the following steps:
+1. Uncomment modules you do not want to run.
+    ```yaml
+    no_gui: true
+    #no_kibana: true
+    #no_floodlight_2: true
+    ```
+    The example above will help you to run modules `kibana` and `floodlight_2` but do not run `gui`.
+2. Run `make update-props` to apply the changes
+3. Build and run Open Kilda as usual.
+
+NOTE: If you will apply these changes on an up and running environment then commands `make clean`/`make clean-test` 
+wouldn't stop modules which were disabled. So it is recommended to apply changed `docker-compose.yaml` file on
```

Comment:
I would recommend to avoid using `So` in the beginning of a sentence (we are overusing `so` and often use it incorrectly. The grammar on this topic is surprisingly complicated). I am not really sure I want to write it 😃, but ok, let's try: "so" here is a filler word and needs to be separated with a comma. At the same time, the meaning of this "so" is a coordinating conjunction (it connects to the previous sentence and feels like "as a result", "that's why") and we cannot start a sentence with a coordinating conjunction. It's ok to start a sentence with "so" when it is a subordinating conjunction.